deepinterpolation/cli/schemas.py

```diff
@@ -0,0 +1,106 @@
+import argschema
+import marshmallow as mm
+import datetime
+
+
+class GeneratorSchema(argschema.schemas.DefaultSchema):
+    """defaults set in this class should be applicable to
+    both ophys and ephys, and both the training generator
+    and test generator.
+    """
+    type = argschema.fields.String(
```

Comment:
Is it possible to fill in the description field for these attributes?

Reply:
It is possible. I was planning to do so after a little more cleaning up of `ClassLoader` etc.

Reply:
done. not 100% complete, but, an improvement for now.